browser/static/js/main_zstack.js

```diff
@@ -1395,26 +1410,35 @@ function start_caliban(filename) {
     handle_mouseup();
    });
 
-  // define image onload cascade behavior
+  load_file(filename);
+
+  // define image onload cascade behavior, need rawHeight and rawWidth first
+  adjust = new ImageAdjustment(width=rawWidth, height=rawHeight)
```

Comment:
Refactor the object's variable name to be a noun, `adjuster` maybe. I would refactor the class to something like `ImageAdjster` too. IMO `ImageAdjustment` sounds like a class that doesn't get instantiated.